src/App.js

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from 'react';
+import './App.css';
+import { useDispatch } from 'react-redux';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Homepage from './pages/Homepage';
+import Stockpage from './pages/Stockpage';
+import { getStocks } from './redux/stocks/stocksSlice';
+
+function App() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getStocks());
+  }, [dispatch]);
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" exact element={<Homepage />} />
+        <Route path="/details/:symbol" exact element={<Stockpage />} />
+      </Routes>
+    </Router>
```

Comment:
- [ ] 👍 Your design is on the right track! 🌟 However, there seems to be a minor issue with overflow, which is causing a vertical scroll to appear. To align your design with the specifications: To address the overflow issue and ensure your design matches the specifications, please consider the following steps:
1. **Overflow Check:** Identify which element or elements are causing the overflow. Use browser developer tools to inspect the elements and detect the source of the issue.
2. **Adjust Content:** Once you've pinpointed the element causing the overflow, adjust its content, size, or margins as needed to ensure it fits within the viewport without causing a vertical scroll. You might need to make layout adjustments, such as adjusting the width, height, or padding of containers.

[Stock-App.webm](https://github.com/AliRezaBmeDu/metrics-webapp/assets/55840999/93ac116c-9388-41e6-bf05-69287e161737)


By implementing these adjustments, you'll resolve the overflow issue and achieve a design that meets the specifications without unnecessary scrolling. 📏🖥️ Keep up the great work! 👏👨‍💻🎨

Reply:
done